### ISSUE TITLE
Create an 'event' to allow a subset of users to undertake a full route.

### ISSUE THREAD

**Issue by charlesbalderson** [JosephRedfern/TheAuditTrail#5]
Currently the entire route is shared for all users. A user should be able to create an 'event' and invite other users to undertake path sections. This will not affect other 'events' sections.

Basically... remove the global sections.

**Comment by JosephRedfern** [comment]
Yep - as discussed. Potential to allow users to register for events without requiring approval, too. Should be an option during event creation.

**Comment by charlesbalderson** [comment]
Would you want an event to be private/unlisted? But then also whether it requires approval to join by the event admin.

I suppose you could have a listed event but you still might want to be able to be in control of who is a member.

**Comment by JosephRedfern** [comment]
(For the initial release, we've decided not to fuss with the private/unlisted stuff)
